docs/usage.md

```diff
@@ -1,103 +1,152 @@
-| [Home](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/release/1.0.1/README.md) | 
-|--------------------------------------------| 
+| [Home](../README.md) |
+|----------------------|
 
 # Usage
 
 ## Playbook Execution Modes
 
-The GDPR playbooks can be executed in 'Test Mode' as well as in 'Production Mode'
+You can execute the GDPR playbooks in *Test Mode* as well as in *Production Mode*
 
-  - The playbooks are by default configured to execute in "Test Mode"
-  - Under the "Test Mode", all the email communication to be towards DPO, Affected Individuals or DPA will be done using the test email ID that is configured for testing while in 'Production Mode' the actual email IDs will be used
-  - In order to change execution mode or test email-ID, follow the below procedure:
-    - Goto to the "Create Data Compliance Record" playbook under the '10 - SP - GDPR Framework' collection and open the 'Configuration' step
-    ![Execution Mode Configuration](res/execution-mode-configuration.png)
-    - To change of playbooks execution mode from "Test Mode" to "Production", Modify value of **testMode** variable to `false`
-    - Likewise, to change test email-ID change value of **testEmail** variable
-    ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parametes.png)
+- The playbooks are by default configured to execute in *Test Mode*
+  - The *Test Mode* uses a test email address for all email communication towards Data Protection Officer (DPO), affected individuals, or Data Protection Authority(DPA)
+  - The *Production Mode* uses actual email addresses  of relevant authorities and stakeholders
 
+### Production Mode
 
-## Gathering Details on Personal Data Breach
+To change the execution mode to *Production* make the following changes:
 
-- When a personal data breach is detected in an incident, choose the value of the **Was Personal Data Affected?** field as **Yes**.
+1. Go to the *Create Data Compliance Record* playbook under the **10 - SP - GDPR Framework** collection and open the *Configuration* step
 
-    ![Personal Data Affected](res/personal-data-affected.png)
+  ![Execution Mode Configuration](res/execution-mode-configuration.png)
 
+2. Modify the value of the `testMode` variable to `false`
+3. Change the value of the `testEmail` variable to a valid email address.
```

Comment:
To move to production mode, user needs to give blank value for the variable instead of giving a valid value.

While in case of test mode user needs to give a valid email address so that all the email should go to the respective email address